Windows/MainWindowMenu.cpp

```diff
@@ -1076,6 +1101,7 @@ namespace MainWindow {
 		CHECKITEM(ID_OPTIONS_SHOWFPS, g_Config.iShowFPSCounter);
 		CHECKITEM(ID_OPTIONS_FRAMESKIP_AUTO, g_Config.bAutoFrameSkip);
 		CHECKITEM(ID_OPTIONS_FRAMESKIP, g_Config.iFrameSkip != FRAMESKIP_OFF);
+		CHECKITEM(ID_OPTIONS_FRAMESKIPTYPE, g_Config.iFrameSkipType);
```

Comment:
Ah, this doesn't work - you'll need a separate one for each of `ID_OPTIONS_FRAMESKIPTYPE_COUNT` and `ID_OPTIONS_FRAMESKIPTYPE_PERCENT` with `g_Config.iFrameSkipType == FRAMESKIPTYPE_COUNT` etc.

-[Unknown]